airbyte-webapp-e2e-tests/cypress/pages/destinationPage.ts

```diff
@@ -1,18 +1,21 @@
-const newDestination = "button[data-id='new-destination'";
-const addSourceButton = "button[data-id='select-source']";
+const newDestination = "button[data-id='new-destination']";
+const addSourceButton = "button[data-testid='select-source']";
 
 export const goToDestinationPage = () => {
   cy.intercept("/api/v1/destinations/list").as("getDestinationsList");
   cy.visit("/destination");
   cy.wait(3000);
-  cy.wait("@getDestinationsList");
 };
 
 export const openNewDestinationForm = () => {
-  cy.get(newDestination).click();
+  cy.wait("@getDestinationsList").then(({ response }) => {
+    if (response?.body.destinations.length) {
+      cy.get(newDestination).click();
+    }
+  });
   cy.url().should("include", `/destination/new-destination`);
 };
 
-export const openAddSource = () => {
+export const aopenAddSource = () => {
```

Comment:
Seems this method got accidentally renamed.

Reply:
This was the issue with the e2e tests failing 🤦‍♀️